pages/_seasons/unit/_unit.vue

```diff
@@ -16,15 +16,31 @@
         <!--        상세 -->
         <div class="main-wrap">
             <div class="unit_top_banner p-relative" :style="`background-image: url(${unitDetail.iconPath})`">
-                <div class="p-absoulte w100p">
+                <div class="p-absoulte w100p over_text">
                     <strong class="text-white">{{ unitDetail.krName }}</strong>
                 </div>
             </div>
+            <div class="unit_content_wrap">
+                <div class="unit_status">
+                    <div class="status_box">
+                        <div class="status_title">체력</div>
+                        <div class="status_value">{{ unitDetail.stats.hp }}</div>
```

Comment:
Codacy has a fix for the issue: [Replace `{{·unitDetail.stats.hp·}}` with `⏎····························{{·unitDetail.stats.hp·}}⏎························`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9701485)
```suggestion
                        <div class="status_value">
```